tests/tests_e3/fs/main_test.py

```diff
@@ -208,10 +209,28 @@ def test_tree_state():
     state6 = e3.fs.get_filetree_state("toto")
     assert isinstance(state6, str)
 
+    with open("toto", "wb") as f:
+        f.write(b"hello world.")
+
     # check that get_filetree_state accept unicode
     state7 = e3.fs.get_filetree_state("toto")
     assert isinstance(state7, str)
```

Comment:
Should an `assert state7 != state6` be added here ?

Reply:
Maybe I can add state7 == state6 here as hash content is false by default

Reply:
but here I'm not sure it is useful here

Reply:
done